docs/index.rst

```diff
@@ -0,0 +1,42 @@
+.. Vulcanexus documentation master file.
+
+.. include:: rst/formalia/titlepage.rst
+
+.. _index_introduction:
+
+.. toctree::
+   :caption: Introduction
+   :maxdepth: 2
+   :hidden:
+
+   /rst/formalia/titlepage.rst
```

Comment:
There is an issue that it is also happening in all our documentation repositories (checked in Fast DDS & Fast DDS Statistics Backend). If in the initial page we click the `Next` button, the same page is shown again. The first is the `index.html` while the second (identical) is the `rst/formalia/titlepage.html`.

```suggestion
```

The issue seem to be solved removing this file. It has been included previously with the `include` directive and thus it has already been shown. As a consequence, in the index, we lose the Introduction entry. Personally I do not think it is really necessary having the link there, because clicking on the logo directly, shows the `index` page.

Reply:
I think having to click on next twice is the lesser evil here. This way, we have the into sections in the TOC. Else, things that will come in the intro such as `contact and support` would get lost.